papers/drivendata/main.md

```diff
@@ -590,3 +610,9 @@ Zamba is designed to accelerate human workflows rather than replace human expert
 ## Conclusion
 
 Zamba is a powerful tool in supporting wildlife conservation. Initially developed to tackle the technically demanding task of processing camera-trap videos, it has since expanded to handle images as well. A key capability of Zamba is the ability to fine-tune models to better target specific habitats or expand to new subjects of interest, with support for species classification for both videos and images. The Zamba open source package supports programmatic use at the command line or as a Python library, while Zamba Cloud provides a no-code option for non-programmer users. The use of automated machine learning workflows for camera trap data can save countless hours of valuable time, speed up conservation interventions, and enable camera traps to be used to their fullest potential.
+
+## Acknowledgments
+
+This project has been supported by funding from the Max Planck Institute for Evolutionary Anthropology, the Arcus Foundation, the Patrick J. McGovern Foundation, and the WILDLABS Awards 2024.
+
+The authors express their gratitude to collaborators, advisors, and data providers at the following organizations: the Pan African Programme: The Cultured Chimpanzee, the Max Planck Institute for Evolutionary Anthropology, the German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig, the Helversen’sche Stiftung, and Dan Morris. The authors thank the other contributors to the Zamba code base: Justin Chung Clark, Robert Gibboni, Katie Wetstone, and Casey Fitzpatrick. The authors also thank Dmytro Poplavskiy, the winner of the Pri-Matrix Challenge, whose winning solution was the basis for the original Zamba V1 video species classification model; and Kirill Brodt, the second-place winner of the Deep Chimpact Challenge, whose solution is the basis for the depth estimation model. Finally, the authors thank the reviewers who provided constructive comments and suggestions to improve the quality of the manuscript.
```

Comment:
PanAf program is within MPI EVA so not sure it needs a separate call out

Tammy also worked on zamba so should be included. I think Isaac worked on ZC and should be included too

Hannah also provided edits on an earlier draft and should be thanked